client/my-sites/comments/comment-list/index.jsx

```diff
@@ -37,22 +40,37 @@ export class CommentList extends Component {
 		} );
 	}
 
-	toggleCommentLike = commentId => {
-		const comments = map( this.state.comments, comment => {
-			if ( commentId === comment.ID ) {
-				const newLike = ! comment.i_like;
-				// If like changes to true, also approve the comment
+	getEmptyMessage = () => {
+		const { status, translate } = this.props;
+
+		switch ( status ) {
+			case 'unapproved':
 				return {
-					...comment,
-					i_like: newLike,
-					status: newLike ? 'approved' : comment.status,
+					title: translate( 'No new comments yet.' ),
+					line: translate( 'Your queue is clear.' ),
 				};
-			}
-			return comment;
-		} );
-
-		this.setState( { comments } );
-	};
+			case 'approved':
+				return {
+					title: translate( 'No approved comments.' ),
+					line: translate( 'Your queue is clear.' ),
+				};
+			case 'spam':
+				return {
+					title: translate( 'No spam comments.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **63** times:
`translate( 'spam comments' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).